src/shared/views/AppLayout/appLayout.css

```diff
@@ -0,0 +1,21 @@
+.main {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.body {
+  flex: 1 0 auto;
+}
+
+.navigation {
+  background-color: #f1f1f1;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.topMenu {
+  height: 34px;
```

Comment:
I believe this is un-needed. Remove this and it should fix the photo going on the wrong place